### DISCUSSION TITLE
Algorithms for numerical methods on differential equations

### DISCUSSION THREAD

**Discussion by juanmagdev** [TheAlgorithms/Python#8174] [Ideas]
Hello everyone,

I would like to propose adding algorithms for various numerical methods used in solving differential equations to GitHub's algorithm repository. The repository currently contains many useful algorithms, but I believe that adding numerical methods algorithms would be of great value to the community.

As a student currently studying maths at university, I am already working on some of these algorithms and believe that sharing them with the community would be beneficial. Some of the commonly used methods include:

 - Euler's method
 - Midpoint method
 - Runge-Kutta methods (2nd, 3rd, and 4th order)
 - Adams-Bashforth methods
 - Multistep methods
 - ...
 
By adding algorithms for these and other numerical methods to the repository, we can provide a valuable resource for anyone interested in learning or implementing these methods in their own work. Additionally, it can foster collaboration and improvement on existing algorithms.

I am willing to contribute to the creation of these algorithms, and I invite others in the community to do the same. Please let me know your thoughts on this proposal, and if there is interest, we can work together to create and submit these algorithms to the repository.

Thank you for your time and consideration.

Best regards,

juanmagdev

**Comment by EngineerMarek** [comment]
That sounds good to me, my professors never show code, only demonstrations, so having some community reviewed implementations would be great!
